crate_universe/private/common_utils.bzl

```diff
@@ -71,11 +71,18 @@ def get_rust_tools(repository_ctx, host_triple):
         cargo_config = repository_ctx.path(repository_ctx.attr.cargo_config)
         repository_ctx.symlink(cargo_config, cargo_home_config)
 
+    if repository_ctx.attr.rust_version.startswith(("beta", "nightly")):
+        channel, _, version = repository_ctx.attr.rust_version.partition("/")
```

Comment:
I'm confused by this line in conjunction with the documentation for rust_version below:
```
The version of Rust the currently registered toolchain is using. Eg. `1.56.0`, or `nightly/2021-09-08
```
This line makes it look that we're going to have a string where partitioning by `/` has 3 components, I'm not sure what the second one is...

Reply:
Where is the 3rd component coming from? [string.partition](https://bazel.build/rules/lib/string#partition) is documented as
> Splits the input string at the first occurrence of the separator sep and returns the resulting partition as a three-element tuple of the form (before, separator, after). If the input string does not contain the separator, partition returns (self, '', ''). 

So of the 3 values being returned, only 2 are expected to contain meaningful information, right?